deployment/aws-python-storage-stack-automation/storage_stack_lifecycle.yaml

```diff
@@ -352,6 +352,202 @@ Resources:
+              - "iam:PassRole"
+              - "iam:DetachRolePolicy"
+              - "iam:GetRole"
+              - "iam:GetRolePolicy"
+              - "iam:DeleteRole"
+              - "iam:DeleteRolePolicy"  
+            Resource: "*"
+          - Effect: "Allow"
+            Action:
+              - "cloudformation:ListExports"
+              - "cloudformation:ListStackInstances"
+              - "cloudformation:DescribeStackResource"
+              - "cloudformation:ListStackResources"
+              - "cloudformation:DescribeStackEvents"
+              - "cloudformation:DescribeStackInstance"
+              - "cloudformation:GetTemplateSummary"
+              - "cloudformation:DescribeStacks"
+              - "cloudformation:GetTemplate"
+              - "cloudformation:DeleteStack"
+              - "cloudformation:ListStacks"  
+            Resource: "*"
+          - Effect: "Allow"
+            Action:
+              - "sqs:ListQueues"
+              - "sqs:GetQueueUrl"
+              - "sqs:ListQueueTags"
+              - "sqs:ReceiveMessage"
+              - "sqs:DeleteMessage"
+              - "sqs:RemovePermission"
+              - "sqs:GetQueueAttributes"
+              - "sqs:ListDeadLetterSourceQueues"
+              - "sqs:DeleteQueue"  
+            Resource: "*"
+          - Effect: "Allow"
+            Action:
+              - "s3:*"
+              - "sns:Unsubscribe"
+              - "sns:SetTopicAttributes"
+              - "sns:GetTopicAttributes"
+              - "sns:RemovePermission"
+              - "sns:DeleteTopic"  
+            Resource: "*"
+          - Effect: "Allow"
+            Action:
+              - "lambda:DeleteFunction"
+              - "lambda:InvokeFunction"
+              - "lambda:GetEventSourceMapping"
+              - "lambda:GetFunction"
+              - "lambda:RemovePermission"  
+            Resource: "*"
+          - Effect: "Allow"
+            Action:
+              - "secretsmanager:GetSecretValue"  
+            Resource: !Ref wsapi
+          - Effect: Allow
+            Action:
+              - "sts:AssumeRole"
+
+            Resource: !GetAtt
+              - implodestoragelambda
+              - Arn
+      Roles:
+        - !Ref implodelambdarole
+  implodestoragelambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Runtime: python3.8
+      Timeout: 120
+      Handler: index.lambda_handler
+      Role: !GetAtt
+        - implodelambdarole
+        - Arn
+      Code:
+        ZipFile: |
+          import json
+          import os
+          import boto3
+          import urllib3
+          from botocore.exceptions import ClientError
+          http = urllib3.PoolManager()
+
+          fss_key = os.environ["WS_API"]
+          get_url = "https://cloudone.trendmicro.com/api/filestorage/external-id"
+          post_url = "https://cloudone.trendmicro.com/api/filestorage/stacks"
+          delete_url = "https://cloudone.trendmicro.com/api/filestorage/stacks/"
```

Comment:
How about `external_id_url` and `stacks_url`?

Reply:
updated to ext_id_url and stacks_url